quiz.py

```diff
@@ -3,4 +3,18 @@
 Author: Name
 Date: YYYY-MM-DD
 Description: A brief explanation of what this program does.
-"""
\ No newline at end of file
+"""
+print ("Multiple-Choice Quiz Game")
+print ("")
+print ("1. Which of these agents role is a Duelist?")
+iso = ("(a) Iso")
+kj = ("(b) Killjoy")
+omen = ("(c) Omen")
+print (iso)
+print (kj)
+print (omen)
```

Comment:
In the context of this problem, I don't think it's necessary to set variables in order to print them. A variable is useful for something that you are going to change later in the program. Will `iso` really be assigned something else later in the program execution? It doesn't look like you intend to do that, so it would suffice to just use `print()` statements to output the choices.